themes/hextra/assets/css/custom.css

```diff
@@ -0,0 +1,13 @@
+/* Custom CSS for Memcached documentation */
```

Comment:
Think the right place to put this is `static/assets/css/custom.css` which will get deployed to `public/assets/css/custom.css` which the theme should pick up

Reply:
That was where my first attempt led me, but it doesn't seem to work without further steps. I think you're right that this is the cleaner solution but it requires deeper knowledge of Hugo than I currently have, and I didn't see an easy way of getting there that wouldn't have likely involved hours more of research.

Would it be acceptable to keep `custom.css` where it is now, and open an issue to address after the docs are delivered?

Reply:
what were the symptoms? it just didn't seem to apply at all? (or what further steps were you expecting?)

Reply:
It doesn't apply at all. If it does apply, then the commands on e.g. features/proxy2/install/#download are preceded with `$ ` prompts.

I think what's missing is whatever magic is necessary to make both of the following happen:

1. Put the custom CSS file in whatever collection of static files that Hugo generates.
2. Link to that file from the <head> sections of all of the generated HTML pages.

Certainly that's possible—it's probably even a basic Hugo trick. But since I'm coming at all of this rather sideways, I don't know it.

Reply:
@jmacdotorg I got this working. steps:

- revert the change from themes/hextra/etc
- put your custom.css file in `assets/css/custom.css` _from the top level of the repository_
ie: the same directory that hugo.yaml is in, make an assets dir, then a css subdir, then put the custom.css in there. keeping the theme/hextra/etc/custom.css blank as per default.

I've been testing locally and editing that assets/css/custom.css file makes the $ change come and go.

Reply:
Excellent work! This works for me too, and I've updated this pull request appropriately. (https://github.com/memcached/memcached.github.io/pull/3/commits/10c845ce41c181ed1b991b8ff409354b4357e3f9)